From f131d2402e54ba090b37201e4674105b330d6375 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Mon, 2 Dec 2019 14:04:25 +0100
Subject: [PATCH] Remove flash tab from LuCI

This removes the link to Flash Operations from LuCI menu and users can not
access it directly as it can be dangerous as we don't support updates or
backups provided by LuCI.

Rebased for OpenWrt 18.06.
Inspiration: https://gitlab.labs.nic.cz/turris/turris-build/merge_requests/65
(Main difference: users can not access it even directly!)
---
 .../luasrc/controller/admin/system.lua                 | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/modules/luci-mod-admin-full/luasrc/controller/admin/system.lua b/modules/luci-mod-admin-full/luasrc/controller/admin/system.lua
index 153615b58..657e1742e 100644
--- a/modules/luci-mod-admin-full/luasrc/controller/admin/system.lua
+++ b/modules/luci-mod-admin-full/luasrc/controller/admin/system.lua
@@ -33,13 +33,13 @@ function index()
 	end
 
 	entry({"admin", "system", "flashops"}, call("action_flashops"), _("Backup / Flash Firmware"), 70)
-	entry({"admin", "system", "flashops", "reset"}, post("action_reset"))
-	entry({"admin", "system", "flashops", "backup"}, post("action_backup"))
-	entry({"admin", "system", "flashops", "backupfiles"}, form("admin_system/backupfiles"))
+	-- entry({"admin", "system", "flashops", "reset"}, post("action_reset"))
+	-- entry({"admin", "system", "flashops", "backup"}, post("action_backup"))
+	-- entry({"admin", "system", "flashops", "backupfiles"}, form("admin_system/backupfiles"))
 
 	-- call() instead of post() due to upload handling!
-	entry({"admin", "system", "flashops", "restore"}, call("action_restore"))
-	entry({"admin", "system", "flashops", "sysupgrade"}, call("action_sysupgrade"))
+	-- entry({"admin", "system", "flashops", "restore"}, call("action_restore"))
+	-- entry({"admin", "system", "flashops", "sysupgrade"}, call("action_sysupgrade"))
 
 	entry({"admin", "system", "reboot"}, template("admin_system/reboot"), _("Reboot"), 90)
 	entry({"admin", "system", "reboot", "call"}, post("action_reboot"))
-- 
2.24.0
